src/Entity/Item.php

```diff
@@ -319,4 +319,16 @@ public function equals( EntityDocument $entity ) {
 		       && $this->statements->equals( $entity->statements );
 	}
 
+	/**
+	 * @see EntityDocument::__clone
+	 *
+	 * @since 5.0
```

Comment:
This shouldn't be called outside of this component so I'd remove the `@since` tag and perhaps mark this somehow as package-private.